build/local_install_or_update.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# Generic setup
+set -e
+
+# Ensure we have a $FLEPI_PATH
+if [ -z "${FLEPI_PATH}" ]; then
+    export USERDIR=$(pwd)
+    export FLEPI_PATH="$USERDIR/flepiMoP"
+    export FLEPI_PATH=$( realpath "$FLEPI_PATH" )
+    echo "Using '$FLEPI_PATH' for \$FLEPI_PATH."
+fi
```

Comment:
So the instructions ask users to run this script from the `flepiMoP` repo so the value of `$USERDIR` is `.../flepiMoP` so the value of `$FLEPI_PATH` is `.../flepiMoP/flepiMoP`. I think maybe the better approach is to get rid of this `$USERDIR` concept in the local script and just take `export FLEPI_PATH=$( pwd )`, maybe can check that the `dirname` of `$FLEPI_PATH` is `flepiMoP` as a catch?